example/vpc-and-vpn/main.tf

```diff
@@ -1,16 +1,12 @@
-terraform {
-  required_version = "0.15.4"
-}
-
-module "vpc-and-vpn" {
+module "vpc_and_vpn" {
     source = "../../modules/vpc-and-vpn"
 
-    name = "fury"
-    
+    name = "fury-example"
```

Comment:
Maybe, we can also add somewhere that this should be equal to the cluster name?

Reply:
Absolutely, I added a comment to the `main.tf` file of both the `vpn-and-vpc` and `eks` modules.